examples/with-passport/README.md

```diff
@@ -4,7 +4,7 @@ This example show how to use [Passport.js](http://www.passportjs.org) with Next.
 
 The example shows how to do a login, signup and logout; and to get the user info using a hook with [SWR](https://swr.now.sh).
 
-A DB is not included. You can use any db you want and add it [here](/lib/user.js).
+A DB is not included. You can use any db you want and add it [here](https://github.com/zeit/next.js/blob/canary/examples/with-passport/lib/user.js).
```

Comment:
I think the following works too:

```suggestion
A DB is not included. You can use any db you want and add it [here](lib/user.js).
```

Reply:
Cool I'll test that out really quick. I tried a few different variations but not that. Thanks!

Reply:
Yep that did it! Worked on my fork. Thanks 🙏